hit_creation_sim.py

```diff
@@ -37,6 +37,26 @@ def create_missions_for_routes(engine, cursor, route_rows):
                 'distance_ft': 1000,
                 'distance_mi': 0.189394
             })
+            mission_rows_to_insert.append({
+                'region_id': region_id,
+                'label': 'mturk-mission',
+                'level': 2,
+                'deleted': False,
+                'coverage': None,
```

Comment:
I need to confirm whether the progress bar even interacts with the coverage attribute for this